src/api/app/assets/javascripts/webui/report.js

```diff
@@ -40,6 +40,28 @@ function hideReportButton(element) {
   $(element).addClass('d-none');
 }
 
+/* exported showYouReportedMessage */
+function showYouReportedMessage(reportLinkId, reportableType, reportableId, message) {
+  switch(reportableType) {
+    case 'Comment':
+      // Comments differ depending on where they are, so this is why we have two ways. If an element isn't found, nothing will happen...
```

Comment:
I won't say this is great. As I wrote in the PR description, given how the comments are rendered, I decided to do it this way.